index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Twitter. It's what's happening.</title>
+    <link rel="stylesheet" href="./master.css">
+    <link href="/Users/KingBrutus/Desktop/Pursuit/Unit2/Pursuit-Core-Web-Twitter-CSS-Assignment/favicon.ico" rel="icon" type="image/x-icon" />
+</head>
+
+<body>
+
+    <main id="container">
+
+        <div id="left">
+        <section id="blue">
+            <div class="list">  
+                <ul>
+                    <li class="itemList" id="search">Follow your interests</li>
+                    <li class="itemList" id="search">Hear what people are talking about.</li>
+                    <li class="itemList" id="search">Join the conversation</li>
+                 </ul>
+            </div>  
+            <div>
+                <img id="birdbackground" src="/Users/KingBrutus/Desktop/Pursuit/Unit2/Pursuit-Core-Web-Twitter-CSS-Assignment/images/birdbackground.png"></img>
+                <!-- <img id="card_wide_blue" src="/Users/KingBrutus/Desktop/Pursuit/Unit2/Pursuit-Core-Web-Twitter-CSS-Assignment/images/card_wide_blue.png"></img> -->
+            </div>
+        </section>
+        </div>
+
+        <div id="right">
+        <section id="white">
+           <div>
+               <form>
+                   <input id="username" type="text" placeholder="Phone, email, or username"/>
+                   <input id="password" type="text" placeholder="password"/>
+                   <button class="login">Log in</button>
```

Comment:
For future reference, log-in or sign-in buttons that are in forms should be of the type submit.